Fix incorrect punctuation in components docs (#1225)
nuxt/nuxtjs.org: content/en/guides/directory-structure/components.md

To access asynchronous data from an API in your components you can use Nuxt [`fetch()` hook](/docs/2.x/features/data-fetching#the-fetch-method).

> Using `$fetchState.pending`, we can show a message when the data is waiting to be loaded. We can also use `$fetchState.error` to show an error message if there is an error fetching the data. When using fetch, we must declare the data in the data property. This then gets filled with the data that comes from the fetch.

Reviewer: ```suggestion
By checking `$fetchState.pending`, we can show a message when data is waiting to be loaded. We can also check `$fetchState.error` and show an error message if there is an error fetching the data. When using `fetch()`, we must declare the appropriate properties in `data()`. The data that comes from the fetch can then be assigned to these properties.
```

    Button.vue
```

The component name will be based on its own filename. Therefore, the component will be:

```html
<button />
```

We recommend you use the directory name in the filename for clarity:

```bash
components/
  base/
    BaseButton.vue
```

> However, if you want to keep the filename as `Button.vue`, then you can use the `prefix` option in the Nuxt config to add a prefix to all components in a specific folder.

Reviewer: ```suggestion
However, if you want to keep the filename as `Button.vue`, then you can use the `prefix` option in `nuxt.config` to add a prefix to all components in a specific folder.
```

}
```

> And now in your template you can use `BaseButton` instead of Button without having to make changes to the name of your `Button.vue` file.

Reviewer: ```suggestion
And now in your template you can use `BaseButton` instead of `Button` without having to make changes to the name of your `Button.vue` file.
```

```html{}[pages/index.vue]
<BaseButton />
```

> <base-alert type="next">
> Learn more about the [components module](/blog/improve-your-developer-experience-with-nuxt-components).
> </base-alert>

Reviewer: ```suggestion
<base-alert type="next">Learn more about the [components module](/blog/improve-your-developer-experience-with-nuxt-components).</base-alert>
```
